.github/workflows/gradle.yaml

```diff
@@ -0,0 +1,33 @@
+name: Java CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - feature/*
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ inputs.platform }} @ ${{ github.event.pull_request.number || github.sha }}'
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@ccb4328a959376b642e027874838f60f8e596de3
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@749f47bda3e44aa060e82d7b3ef7e40d953bd629
```

Comment:
ditto

Reply:
@kaidokert is this also v1.0.6?

Reply:
No i don't know what version it was, lets pick latest from https://github.com/gradle/gradle-build-action/releases

Reply:
updated to v2.4.2, is it accurate though?

Reply:
It is, v2.4.2 SHA=749f47b